plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/context/lightweight-seqd.contexts

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<contexts:Context xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:constraints="http://www.eclipse.org/papyrus/constraints/0.9" xmlns:contexts="http://www.eclipse.org/papyrus/properties/contexts/0.9" xmi:id="_70QJkKDJEeSZxfCXzZz3-w" name="Lightweight Sequence Diagram" label="Sequence Diagram">
+  <tabs xmi:id="_70QJkaDJEeSZxfCXzZz3-w" label="Sequence Diagram" id="lw-seqd" category="org.eclipse.papyrus.seqd" image="" priority="8">
+    <!-- if it shouldn't replace the UML tab, don't use the same section names as in UML.ctx; I postfixed them with 1 -->
+    <sections xmi:id="_70QJkqDJEeSZxfCXzZz3-w" name="SingleMessage1" sectionFile="ui/SingleMessage.xwt">
+      <widget href="ui/SingleMessage.xwt#/"/>
+    </sections>
+    <sections xmi:id="_70QJk6DJEeSZxfCXzZz3-w" name="MultipleMessage1" sectionFile="ui/MultipleMessage.xwt">
+      <widget href="ui/MultipleMessage.xwt#/"/>
+    </sections>
+    <sections xmi:id="_70QJkqDJEeSZxfCXzZz3-l" name="SingleLifeline1" sectionFile="ui/SingleLifeline.xwt">
+      <widget href="ui/SingleLifeline.xwt#/"/>
+    </sections>
+  </tabs>
+  <views xmi:id="_70idvaDJEeSZxfCXzZz3-w" name="SingleMessage" sections="_70QJkqDJEeSZxfCXzZz3-w" automaticContext="true">
```

Comment:
Should the views also have distinct names, to match the sections?

Reply:
Good idea, I'll rename them.